src/main/scala/io/iohk/ethereum/consensus/Ethash.scala

```diff
@@ -8,6 +8,7 @@ import io.iohk.ethereum.crypto.{kec256, kec512}
 import io.iohk.ethereum.utils.ByteUtils._
 import org.spongycastle.util.BigIntegers
 import org.spongycastle.util.encoders.Hex
+import java.lang.Integer.remainderUnsigned
```

Comment:
😮 
Great find, maybe there are other functions we can replace with imports? 😛 

Have you compared both impls wrt functionality and speed?

Reply:
As terms of seed, I only compared it globally (run ethash with our own implementation and with this import) and there was no speed difference and recording showed almost the same time in `fnv` function which uses it. 
As for other imports, I think there weren't any to use in `ethash` and I have not looked at it globally.